src/api/activities.api.ts

```diff
@@ -0,0 +1,12 @@
+import { baseURL } from "../config";
+import { ActivityItem } from "./activities.api.types";
+
+export async function getAllActivities(): Promise<{data: ActivityItem[]; error: any}> {
+    try {
+      const response = await fetch(baseURL + "activities_get")
```

Comment:
Maybe the string concatenation can be bit more readable (getting rid of the + operatir) with "template string":
`baseURL + "activities_get"` => `` `${baseURL} activities_get` ``

Reply:
Changed.